### ISSUE TITLE
Following the Docs does not start the server

### ISSUE THREAD

**Issue by michaelwhitford** [coast-framework/coast#60]
I followed the docs, and run `make server`, but no server starts.  When I run the repl and try to start the server, it tells me (-main) does not exist.

I'm pretty new to clojure, but following your directions exactly from the documentation website I get this:

```
make server
clj -m server
Clojure 1.10.0
user=>
```

```
make repl
clj -R:repl bin/repl.clj
Clojure 1.10.0
user=> (in-ns 'server)
#object[clojure.lang.Namespace 0x6569dded "server"]
server=> (-main)
Syntax error compiling at (REPL:1:1).
Unable to resolve symbol: -main in this context
```

**Comment by swlkr** [comment]
Hmm, I also gave it a shot on a new project with a new `.m2` folder, a new `coast` shell script in `/usr/local/bin` and it worked…

https://asciinema.org/a/239625

**Comment by michaelwhitford** [comment]
I tried it again with everything installed from scratch.  I removed clojure, the .m2 directory, and the coast helper script and installed it all from scratch.  I had the same result, the server does not start.  Is it my version of Java perhaps?
```
java -version
openjdk version "10.0.2" 2018-07-17
OpenJDK Runtime Environment (build 10.0.2+13-Ubuntu-1ubuntu0.18.04.4)
OpenJDK 64-Bit Server VM (build 10.0.2+13-Ubuntu-1ubuntu0.18.04.4, mixed mode)
```
```
clj --help
Clojure 1.10.0
user=> (clojure-version)
"1.10.0"
```

**Comment by swlkr** [comment]
I tried it with `openjdk 10.0.2` and `clojure v.1.10.0` and it still worked in both the repl and `make server`.

Although to get to the repl I just ran `clj` in the project directory with no arguments:

```bash
~/Projects/test-project clj
Clojure 1.10.0
```

And once it was running I required the `server` namespace and started the server that way

```clojure
user=> (require '[server])
nil
user=> (server/-main)
Server is listening on port 1337
nil
user=> 
```

**Comment by swlkr** [comment]
I'm also on macOS if that makes a difference

**Comment by michaelwhitford** [comment]
I have tried many different things to get this to work, but it still will not start the server, and I cannot do it manually because the repl tells me the server namespace does not exist.  I am running Linux Mint 19.1 which is based on Ubuntu 18.04.  I have not had clojure issues with other libraries I have played with, and have to pass on coast at this point because I need to get working on my project immediately.  I'll give it another try in the future and hopefully it will work with my setup.
